start-project-template-inventory.json

```diff
@@ -0,0 +1,64 @@
+{
+    "bff-with-react": {
+        "coreTemplates": [ 
+            { "name": "im-build-dotnet-ci.yml", "canHaveMultiples": false},
+            { "name": "im-deploy-az-app-manually.yml", "canHaveMultiples": true},
+            { "name": "im-deploy-multiple-items-at-once.yml", "canHaveMultiples": false},
```

Comment:
This question will likely be answered when I see the start-project side of things, but I'm guessing having these two be "core" templates means users will have to remove one if they don't need it? I don't think that's necessarily a bad thing, I'm just wondering.

Reply:
The core templates will get put in their .github/workflows folder without any input.  The ones that are optional will only be included if they say yes, include those.  The core/optional follow what was laid out in git-er-done on the [recommended workflows](https://github.com/im-practices/git-er-done/blob/main/actions/recommended-templates.md#terraform-core-templates) page.

In this case the im-deploy-multiple-items-at-once.yml will probably be removed by most teams because they don't want to deploy all of their things at the same time.  That one is a bit unusual but it's kind of a "core" thing for people with that scenario.